strong_password_detection.py

```diff
@@ -0,0 +1,42 @@
+def strong_password_detection(password):
+    AllowedSymbols = ["#", "@", "$", "_", "*", "-"]
```

Comment:
We require type hints and doctests...  Please add tests to cover all the cases below.
```suggestion
def strong_password_detection(password: str) -> bool:
    """
    >>> strong_password_detection("str0ng@Password")
    True
    >>> strong_password_detection("str0ngPassword")
    False
    """
    allowed_symbols = {c for c in "#@$_*-"}
```